docs/dev/docs/__manifest__.py.rst

```diff
@@ -169,3 +106,7 @@ Check if some sytem application exists::
 
 See also: :doc:`External dependencies in odoo<../py/external-imports>`
 
+
+IT-Projects' Template
+=====================
+* https://gitlab.com/itpp/handbook/blob/master/technical-docs/__manifest__.md
```

Comment:
It's not a template, actually. It's a link to guidelines for now. I'm not sure whether we will have  template instuction on that link or not